ArkhamDisplay/CityWindow.xaml

```diff
@@ -169,10 +169,31 @@
                 VerticalAlignment="Top"
                 Text="0/400" />
 
+            <TextBox
+                Name="SearchBox"
+                Grid.Row="0"
+                Width="250"
+                Height="20"
+                Margin="5,85,0,0"
+                HorizontalAlignment="Left"
+                VerticalAlignment="Top"
+                TextChanged="SearchBox_TextChanged" />
+
+            <Button
+                Grid.Row="0"
+                Width="20"
+                Height="20"
+                Margin="257,85,0,0"
+                HorizontalAlignment="Left"
+                VerticalAlignment="Top"
+                Click="Button_Click">
+                X
+            </Button>
```

Comment:
I don't _hate_ the idea of a search box... however, I think there's a larger conversation to be had about how exactly such a feature would be used before we go ahead and implement it.

You've also only implemented it for a single game.